src/test/java/uk/gov/pay/webhooks/webhook/WebhookServiceTest.java

```diff
@@ -30,7 +31,7 @@
     private final WebhookDao webhookDao = mock(WebhookDao.class);
     private final EventTypeDao eventTypeDao = mock(EventTypeDao.class);
     private final InstantSource instantSource = InstantSource.fixed(Instant.now());
-    private final ExternalIdGenerator externalIdGenerator = new ExternalIdGenerator();
+    private final IdGenerator idGenerator = mock(IdGenerator.class);
```

Comment:
I was going to say that we usually use `@Mock` annotations but the other mocks in this class don’t, so carry on.